src/core/renderController.ts

```diff
@@ -22,8 +22,8 @@ export module Core {
    * ```
    */
   export module RenderControllers {
-    var _componentsNeedingRender: {[key: string]: Component} = {};
-    var _componentsNeedingComputeLayout: {[key: string]: Component} = {};
+    var _componentsNeedingRender = new Utils.Set<Component>();
+    var _componentsNeedingComputeLayout = new Utils.Set<Component>();
```

Comment:
Not part of this CR, but I thought we decided (or was brought up during the API refactor) that we instantiate all instance variables in the constructor as opposed to the declaration. Are we still iterating on that idea?